storage-proofs/src/stacked/proof.rs

```diff
@@ -1300,7 +814,7 @@ mod tests {
 
     table_tests! {
         prove_verify_fixed{
-           prove_verify_fixed_32_4(8 * 32);
+           prove_verify_fixed_32_4(4);
```

Comment:
Commenting here, since GitHub won't let me add a review comment not on a changed line:

Can we add a negative test which shows that a `MultiProof` whose individual proofs are all correct but which doesn't have enough total proofs does not verify? Example: two partitions are required but only one is supplied. The logic enforcing this can be tricky, and I think we need to test it directly.

Reply:
Yes, but lets do this as a follow up, this diff is already very large